.flake8

```diff
@@ -1,4 +1,4 @@
 [flake8]
 exclude = abichecker
 max-line-length = 100
-ignore = E501,F401,E128,E251,E201,E202,E302,E305,F841,E261,E712,E126,E711,E125,E123,E101,E124,E127,E701,E714,W504,E129,E741
+ignore = W503,W504,E501,F401,E128,E251,E201,E202,E302,E305,F841,E261,E712,E126,E711,E125,E124,E127,E701,E714,E129,E741
```

Comment:
W503 is a warning and the warning says that it's actually wrong to warn about it and flake will soon warn about the opposite - so let's ignore it :)